### PR TITLE
#170893840 Design dashboard home page

### PR DESCRIPTION
**Pull request by nziokaivy**
#### What does this PR do?

This pull request sets out to design the home page in the dashboard once a user is logged in.

#### Description of Task to be completed?
-  [x] Add dashboard layout
- [x] Style dashboard component
- [x] Add tests
- [x] Set up the redux structure for the most travelled destination
- [x] Set up the redux structure to get the number of requests made by the user
- [x] Set up the redux structure to get the number of the bookings made by the user

#### How should this be manually tested?

- Clone this repository using the command `git clone https://github.com/andela/team-odd-bn-frontend.git`
- Then navigate into the project `cd team-odd-bn-frontend`
- Switch to the avail requests branch using the command: `git checkout ft-design-homepage-170893840`
- Run the command `npm i `to install all the packages
- Run `npm run dev` to run the server
- Then finally, navigate to the dashboard home page by navigating to the route `/dashboard`

#### What are the relevant pivotal tracker stories?
[170893840](https://www.pivotaltracker.com/story/show/170893840)

##### Mobile View
image
image
##### Desktop View
image

### CODE REVIEW

**Review by hezronkimutai — CHANGES_REQUESTED** [review]
Nice job @nziokaivy , I think you should make the highlighted div to be at the center or make the div be 100% width so that it looks uniform with the other divs.
Screen Shot 2020-01-28 at 15 39 02

**Review by NiyongaboEric — CHANGES_REQUESTED** [review]
Nice design @nziokaivy. I recommend you to increase the most travelled destination font size and remove Home title.
Screen Shot 2020-01-28 at 19 18 54

File: src/__tests__/actionTests/getAllBookingActionTests.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    moxios.uninstall(apiCall);
+  });
+
+  it('Should dispatch GET_ALL_BOOKINGS_SUCCESS', async () => {
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: {
+          message: 'Success',
```
Comment (victkarangwa) [inline src/__tests__/actionTests/getAllBookingActionTests.js]: Would mind placing these data under mocks?

File: src/__tests__/actionTests/getAllBookingActionTests.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        },
+      });
+    });
+    const expectedActions = [
+      {
+        payload: {
+          message: 'Error',
+        },
+        type: GET_ALL_BOOKINGS_ERROR,
+      }];
```
Comment (victkarangwa) [inline src/__tests__/actionTests/getAllBookingActionTests.js]: The same here, I think you can make your code DRY by placing them under mocks data

**Review by victkarangwa — COMMENTED** [review]
Nice work @nziokaivy 
We've been waiting for this dashboard for so long. Thanks for your diligence.
Just have a look at the feedback I left.

**Review by hezronkimutai — APPROVED** [review]
Nice work @nziokaivy

**Review by NiyongaboEric — APPROVED** [review]

**Review by victkarangwa — APPROVED** [review]
Good Job @nziokaivy!

**Review by shemaeric — APPROVED** [review]

**Pull request merged**